lib/dateTime.js

```diff
@@ -187,34 +172,42 @@ function processPattern(pattern, replacements) {
 }
 
 export function convertUTCToLocalDateTime(date) {
-	const timezone = getDocumentLocaleSettings().timezone.identifier;
-	return convertDateToTimezone(date, timezone);
+	if (!getDocumentLocaleSettings().timezone.identifier) {
+		return date;
+	}
+	const utcDate = new Date(Date.UTC(date.year, date.month - 1, date.date, date.hours, date.minutes, date.seconds));
+	const formattedDateTime = formatDateString(utcDate);
+	const splitDateTime = formattedDateTime.split(' ');
+	const splitDate = splitDateTime[0].split('/');
+	const splitTime = splitDateTime[1].split(':');
```

Comment:
Another option here that _might_ be faster would be to use a Regex and bracket groups inside to extract the different parts. This is wrong, but for example something like:

```
const re = /([0-9]{1,2})\/([0-9]{1,2})\/([0-9]{4}) ([0-9]{1,2}):([0-9]{2}):([0-9]{2})/;
const match = formattedDateTime.match(re);
if (match === null || match.length !== 7) {
    // uh... not sure
}
return {
		month: match[1],
		date: match[2],
		year: match[3],
		hours: match[4],
		minutes: match[5],
		seconds: match[6]
	};
```;